src/assets/styles/index.css

```diff
@@ -1,568 +1,564 @@
-  margin-bottom: 20px;
-}
-
-.exp-points {
-  margin-bottom: 8px;
-}
-
-/*service card*/
-
-.service-card {
-  border-radius: 6px;
-  background-color: #ffffff;
-  width: 170px;
-  height: 170px;
-  border: 1px solid #1c64f2;
-  box-shadow: 0px 10px 20px rgba(0, 0, 0, 0.04), 0px 2px 6px rgba(0, 0, 0, 0.04),
-    0px 0px 1px rgba(0, 0, 0, 0.04);
-  padding: 12px;
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  align-items: center;
-}
-
-.service-text {
-  font-family: Poppins;
-  font-style: normal;
-  font-weight: 600;
-  font-size: 18px;
-  line-height: 24px;
-  text-align: center;
-  text-transform: capitalize;
-  color: #1b1f3b;
-  margin-top: 20px;
-}
-
-.service-card-icon {
-  width: 48px;
-  height: auto;
-  color: #1c64f2;
-}
-
-/*project card*/
-
-.project-panel {
-  width: 379px;
-  height: 428px;
-  background: #ffffff;
-  border-radius: 6px;
-}
-
-.project-image img {
-  border-radius: 6px 6px 0px 0px;
-  width: 379px;
-  height: 189px;
-  object-fit: cover;
-}
-
-.project-about {
-  padding: 16px 16px;
-}
-
-.project-title {
-  font-family: Poppins;
-  font-weight: 600;
-  font-size: 16px;
-  color: #1b1f3b;
-  margin: 0px;
-}
-
-.project-details {
-  font-style: normal;
-  font-weight: normal;
-  color: #333a68;
-}
-
-.project-stack {
-  font-weight: 300;
-  font-size: 12px;
-  color: rgba(27, 31, 59, 0.8);
-  margin-top: 16px;
-}
-
-.project-stack span:nth-child(2) {
-  margin-left: 8px;
-}
-
-.project-links {
-  font-weight: 300;
-  color: #1b1f3b;
-  display: flex;
-  margin-top: 24px;
-}
-
-.project-links div:nth-child(2) {
-  margin-left: 24px;
-}
-
-.project-icon {
-  color: #1b1f3b;
-}
-
-.project-link {
-  display: flex;
-}
-
-.project-link a {
-  font-family: Poppins;
-  color: #1b1f3b;
-  margin-left: 8px;
-  text-decoration: none;
-  text-decoration-color: #1c64f2;
-}
-
-.project-link a:visited {
-  color: #1b1f3b;
-}
-
-.project-link a:hover {
-  color: #3f83f8;
-}
-
-/*client review*/
-
-.client-says {
-  min-width: 315px;
-  max-width: 320px;
-  min-height: 268px;
-  padding: 16px;
-}
-
-.client-name {
-  display: flex;
-  flex-direction: row;
-  justify-content: flex-start;
-  align-content: space-between;
-  font-weight: 500;
-  font-size: 16px;
-}
-
-.client-image img {
-  width: 45px;
-  height: 45px;
-  border-radius: 6px;
-  object-fit: cover;
-}
-
-.client-details {
-  display: flex;
-  flex-direction: column;
-  margin-left: 8px;
-  justify-content: center;
-}
-
-.review {
-  font-style: normal;
-  font-weight: normal;
-  font-size: 12px;
-  line-height: 23px;
-  color: #000000;
-}
-
-.review p:nth-child(1) {
-  margin-top: 4px;
-}
-
-.name {
-  text-transform: capitalize;
-  font-style: normal;
-  font-weight: 500;
-  font-size: 14px;
-  line-height: 16px;
-  color: #1b1f3b;
-}
-
-.company-name {
-  font-style: normal;
-  font-weight: normal;
-  font-size: 12px;
-  line-height: 14px;
-  color: rgba(27, 31, 59, 0.8);
-  margin-top: 4px;
-}
-
-/* .star-icon img {
+    /*profile card*/
```

Comment:
why are you adding an indent here ? Also install this extension and turn on format on save so that it gets formatted each time you save automatically https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode